### PR TITLE
Generate information about people in teams API

### PR DESCRIPTION
**Pull request by fee1-dead**
cc @pietroalbini

This is for unblocking https://github.com/rust-lang/simpleinfra/pull/110.

### CODE REVIEW

**Comment by fee1-dead** [comment]
oh it doesn't really unblock that PR, I will add something like people.json

**Comment by pietroalbini** [comment]
Changes look good to me, but GitHub is reporting merge conflicts.

**Comment by fee1-dead** [comment]
ok github is not good for resolving merge conflicts. Will resolve it in a moment

**Review by pietroalbini — APPROVED** [review]

**Comment by pietroalbini** [comment]
Code changes still look good! Can someone from core merge this?

File: src/schema.rs [hunk 1]
```diff
@@ -57,6 +57,15 @@ pub(crate) enum Email<'a> {
     Present(&'a str),
 }
 
+impl From<Email<'_>> for Option<String> {
```
Comment (Mark-Simulacrum) [inline src/schema.rs:60]: This feels like a pretty unfortunate From impl, can we use a named method here?

**Review by Mark-Simulacrum — CHANGES_REQUESTED** [review]

**Review by Mark-Simulacrum — APPROVED** [review]

**Pull request merged**